tests/mongodb/config.json

```diff
@@ -1,6 +1,6 @@
 {
-    "app_id": "auth-integration-tests-denli",
-    "config_version": 20180301,
+    "app_id": "auth-integration-tests-bhnql",
+    "config_version": 20200603,
```

Comment:
We should really migrate this to the [`realm-app-importer`](https://github.com/realm/realm-js/tree/master/packages/realm-app-importer) to avoid this being constantly overridden.